src/modules/iasql_functions/rpcs/iasql_begin.ts

```diff
@@ -33,9 +34,10 @@ export class IasqlBegin extends RpcBase {
   call = async (
     _dbId: string,
     _dbUser: string,
-    _ctx: Context,
+    ctx: Context,
   ): Promise<RpcResponseObject<typeof this.outputTable>[]> => {
-    const message = 'Transaction started';
+    const currentTransactionId = await getCurrentTransactionId(ctx.orm);
+    const message = `Transaction ${currentTransactionId} started`;
     return [{ message }];
```

Comment:
Nit: We should also return the `transaction_id` as a second column so it is more easily joinable/usable within their own logic, like placing it into a pl/pgsql variable to reference for other calls.